Complete overhaul of cookbook

Sauraus opened the pull request:
- Install SonarQube 5.0.1
- Update sonar.properties for 5.x
- Added Percona DB support
- rename local_database to postgresql recipe
- Migrated to Berks 3.x
- Install java runtime as part of Sonar server install

Sauraus committed:
Complete overhaul of cookbook

- Install SonarQube 5.0.1
- Update sonar.properties for 5.x
- Added Percona DB support
- rename local_database to postgresql recipe
- Migrated to Berks 3.x
- Install java runtime as part of Sonar server install

yurinnick commented on .kitchen.yml:
```diff
@@ -0,0 +1,16 @@
+suites:
+  - name: default
+    run_list:
+      - recipe[sonar::default]
```
Please fix recipe list to get sonar running, sonar::default recipe is now empty

Sauraus replied:
Sorry about that I use Berkshelf mostly and neglected the Kitchen.

I am off to Chef Conf 2015 later today so updates may be a few days late. ;)

yurinnick replied:
Also I could recommend you to try to use kitchen-docker, which could really be much faster then vagrant, but I am not really sure how stable it is.

Sauraus commented:
I am also adding the option to configure the location of data/temp&logs directories to the cookbook right now.

And I am moving the server recipe to use the 'ark' cookbook instead of unzip etc.

yurinnick commented:
It's really cool! Looking forward for your updates.

Sauraus committed:
Make data/temp & logs directories configureable

Install Sonar using 'ark' cookbook

Sauraus committed:
Add nginx proxy recipe

Sauraus commented:
I am currently at Chef Conf and can't run the entire Kitchen suite, but single VMs are coming up just fine using vagrant & berkshelf.

Sauraus committed:
Initial commit for sonarqube cookbook

Commit:
Safely handle sonar jdbc password

- This can cause some error messages in the sonar.log when the password
is modified on a chef-client run before the sonarqube service is
restarted to pickup the updated password.

Commit:
Fix string replacement on postgresql recipe

Commit:
Fix up cookbook naming
